app/backend/src/couchers/servicers/search.py

```diff
@@ -461,6 +461,19 @@ def UserSearch(self, request, context):
                         (1000 + request.search_in_area.radius) / 111111,
                     )
                 )
+            if request.HasField("search_in_viewport"):
+                statement = statement.where(
+                    func.ST_Within(
+                        User.geom,
+                        func.ST_MakeEnvelope(
+                            xmin=request.search_in_viewport.lng_min,
```

Comment:
I think you need to pass them in as positional arguments. I don't think postgres does keyword args!

Reply:
Oh yeah, realized that too 😅 The PR was/is not really ready